anomdetect/src/main/java/com/expedia/adaptivealerting/anomdetect/holtwinters/SeasonalityTypeParser.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2018 Expedia Group, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.expedia.adaptivealerting.anomdetect.holtwinters;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+public class SeasonalityTypeParser {
```

Comment:
Java enums come with this already. See `Enum.valueOf()`.

Reply:
You're right, I didn't do enough whittling down of the original code. Thanks!